File: RpgShell/src/main/java/com/tp/rpg/armors/Helmet.java
package com.tp.rpg.armors;

public class Helmet implements Armor {

    int armorPoints = 10;

    public int reduceDamage(int startingDamage) {
        return startingDamage - armorPoints;
    }

    public int getArmorPoints() {
        return armorPoints;
    }

}

File: RPG/RPG/Classes/Armor/Helmet.cs
using System;
namespace RPG.Classes.Armor
{
    public class Helmet : Armor
    {
        public Helmet()
        {
        }

        public override int ReduceDamage(int incomingDamage)
        {
            return incomingDamage - 1;
        }

    }
}
